resources/views/welcome.blade.php

```diff
@@ -7,7 +7,7 @@
     <h1 class="display-4">Christmas Queue 2019</h1>
```

Comment:
```suggestion
    <h1 class="display-4">Christmas Queue 2020</h1>
```
might wanna change it here as well 🤔